### PR TITLE
Release 2019.2

### PR DESCRIPTION
**Pull request by jlebon**
It's been a while, and there's quite a few fixes and minor enhancements that would be nice to get out.

### CODE REVIEW

**Comment by jlebon** [comment]
There are a few pending PRs that would be nice to get in before this:

https://github.com/ostreedev/ostree/pull/1831
https://github.com/ostreedev/ostree/pull/1841

File: src/libostree/libostree-released.sym [hunk 1]
```diff
@@ -542,6 +542,10 @@ LIBOSTREE_2018.9 {
   ostree_repo_get_default_repo_finders;
 } LIBOSTREE_2018.7;
 
+/* No new symbols in 2019.1 */
+
+/* No new symbols in 2019.2 */
```
Comment (rfairley) [inline src/libostree/libostree-released.sym]: Just remembered from previous work, that `ostree_repo_get_bootloader` was added as a public API in `2019.2`. (I should have added it to `-devel` at the time, seeing the `-devel` file now). Diff where it was added: https://github.com/ostreedev/ostree/pull/1814/files#diff-23d4c8887c7dda91613c9451bf3fe7c0R6107

Comment (jlebon) [reply src/libostree/libostree-released.sym]: Thanks! I updated it.

**Comment by cgwalters** [comment]
@rh-atomic-bot r+ 50876a2

**Comment by jlebon** [comment]
Bahh, needed to add `ostree_repo_get_bootloader` to the apidoc as well.

**Comment by rfairley** [comment]
Ah sorry, I should have caught that when grepping the other API functions. LGTM now!

**Comment by rfairley** [comment]
@rh-atomic-bot r+ 809b989

**Pull request closed without merging**